src/components/Tags/Tags.tsx

```diff
@@ -56,29 +57,28 @@ export interface TagsProps {
   label: string;
 }
 
-const Tags = ({ id, index, className, onRemove, label }: TagsProps) => {
+export const Tags = ({ id, className, onRemove, label }: TagsProps) => {
   return (
     <div className={classnames(`${px}-tag`, `${px}-button`, className)} aria-label="Close Tag">
-      <>
-        <div className={`${px}-tag-label`}>{label}</div>
-        <div
-          onClick={() => onRemove(label)}
-          className={`${px}-close-button`}
-          data-testid={`${id}-item-${index}-close-button`}
-        >
-          <IconButton className={`${px}-tag-x`}>
-            <span>X</span>
-          </IconButton>
-        </div>
-      </>
+      <div className={`${px}-tag__label`}>{label}</div>
+      <div
+        onClick={() => onRemove(label)}
+        className={`${px}-close-button`}
+        data-testid={`${id}-item-close-button`}
+      >
+        <IconButton className={`${px}-close-button__close`}>
+          <CloseIcon />
+        </IconButton>
+      </div>
     </div>
   );
 };
 
-const TagsList = ({ className, tagClassName, onRemove, onClear, tagsList = [], ...props }: TagsListProps) => {
+const TagsList = ({ className, children, tagClassName, i18n = { clearAllLabel: 'Clear All' }, onClear, ...props }: TagsListProps) => {
```

Comment:
the `i18n` default here actually won't work well because it the user ever passes an `i18n` object (for other future labels) WITHOUT a `clearAllLabel` prop, it will override and leave it undefined.